recipes-support/trousers/trousers_0.3.15.bb

```diff
@@ -0,0 +1,8 @@
+# 0.3.14 unpacks to the current directory instead of to a sub-directory
```

Comment:
Is this version-specific comment still needed?

Reply:
Removed